src/encoded/schemas/file.json

```diff
@@ -225,6 +228,60 @@
         },
         "lab.display_title": {
             "title": "Lab"
+        },
+        "file_size" : {
+            "title" : "File Size",
+            "description" : "Filter using range of size of the file",
+            "aggregation_type" : "range",
+            "ranges" : [
+                { "to" : 1024 },
+                { "from" : 1024, "to" : 10240 },
+                { "from" : 10240, "to" : 102400 },
+                { "from" : 102400, "to" : 1048576 },
+                { "from" : 1048576, "to" : 10485760 },
+                { "from" : 10485760, "to" : 104857600 },
+                { "from" : 104857600, "to" : 1073741824 },
+                { "from" : 1073741824, "to" : 10737418240 },
+                { "from" : 10737418240, "to" : 107374182400 },
+                { "from" : 107374182400 }
+            ],
+            "disabled" : true,
```

Comment:
`disabled` is now used in place of `hide_from_view`, though "hide_from_view" is still supported by front-end if want to include agg for e.g. `format=json`. `disabled` will exclude this facet from being aggregated by ElasticSearch in first place, which should improve performance a smidge.